coderd/httpmw/organizationparam.go

```diff
@@ -76,7 +76,7 @@ func ExtractOrganizationMemberParam(db database.Store) func(http.Handler) http.H
 				UserID:         user.ID,
 			})
 			if errors.Is(err, sql.ErrNoRows) {
-				httpapi.Write(rw, http.StatusForbidden, httpapi.Response{
+				httpapi.Write(rw, http.StatusNotFound, httpapi.Response{
```

Comment:
reintroduces information leak about existence of organizations / users because we can return 403 if they exist.